src/targets/common.ini

```diff
@@ -74,6 +74,7 @@ extra_scripts =
 	${env.extra_scripts}
 	pre:python/build_html.py
 lib_deps =
+	makuna/NeoPixelBus @ 2.6.9
```

Comment:
This is not needed for every target. No point to download it for every ESP82xx targets...

Reply:
I'd rather have all the libraries defined in `common.ini` so we only have one place to manage dependency versions.
We could have a library section and list them all independently so only the ones needed for a target are included.
I think that should be a seperate PR if we do it.

Reply:
Agreed, still no point to download it for every env. We can define own section for libs and use custom names